src/legacy/ui/public/vis/editors/default/_agg_params.scss

```diff
@@ -10,3 +10,12 @@
   color: $textColor;
   background-color: $backgroundColor;
 }
+
+.visEditorAggParam--inline {
+  display: inline-block;
+  width: 50%;
+}
+
+.visEditorAggParam--inline-size {
+  padding-left:  $vis-editor-agg-editor-spacing;
```

Comment:
We can start phasing this custom variable out and just use EUI vars. Also, this should be margin so it doesn't adjust the sizing: `margin-left: $euiSizeS`